src/pages/SignInPage.tsx

```diff
@@ -0,0 +1,120 @@
+import * as React from 'react';
```

Comment:
I kept them on purpose in case we want to add a feature for forgotten passwords in the future. However, I think it's better to remove them for now. It won't be complicated to add them later if needed. so, I've just removed them